flink-connectors/flink-connector-kafka-0.8/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer08.java

```diff
@@ -221,7 +221,8 @@ private FlinkKafkaConsumer08(
 				getLong(
 					checkNotNull(props, "props"),
 					KEY_PARTITION_DISCOVERY_INTERVAL_MILLIS, PARTITION_DISCOVERY_DISABLED),
-				!getBoolean(props, KEY_DISABLE_METRICS, false));
+				!getBoolean(props, KEY_DISABLE_METRICS, false),
+				getBoolean(props, KEY_CHECK_UNAVAILABLE_TOPICS, false));
```

Comment:
Should this be named KEY_CHECK_UNAVAILABLE_PARTITIONS ?

Reply:
You're right, I'll change it